learn/fine_tuning_results/faceted_search.mdx

```diff
@@ -149,6 +149,89 @@ The following response shows the lowest and highest book ratings when searching
 
 If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object. Since `rating` was the only numeric facet in our example, it is the only facet returned in the `facetStats` object.
 
+## Sorting facet values
+
+By default, all facet values are sorted in ascending alphanumeric order. You can change this using the `sortFacetValuesBy` property of the [`faceting` index settings](/reference/api/settings#faceting):
+
+<CodeSamples id="faceted_search_2" />
+
+The above code sample sorts the `genres` facet by descending value count. If you searched the `books` index for `classic` using the new settings, you would get the following response:
+
+```json
+{
+  …
+  "facetDistribution": {
+    "genres": {
+      "Fiction": 8,
+      "Literature": 7,
+      "Classics": 6,
+      "Novel": 2,
+      "Horror": 2,
+      "Fantasy": 2,
+      "Victorian": 2,
+      "Vampires": 1,
+      "Tragedy": 1,
+      "Satire": 1,
+      "Romance": 1,
+      "Historical Fiction": 1,
+      "Coming-of-Age": 1,
+      "Comedy": 1
+    },
+   …
+   }
+}
+```
+
+## Searching facet values
+
+Once you have configured [filterableAttributes](/reference/api/settings#filterable-attributes), you can also search for facet values with the [facet search endpoint](/reference/api/facet_search). Facet search supports [prefix search](/learn/advanced/prefix) and [typo tolerance](/learn/configuration/typo_tolerance).
+
+<Capsule intent="Facet search on numbers">
+Internally, Meilisearch represents numbers as [`float64`](https://en.wikipedia.org/wiki/Double-precision_floating-point_format). Since they lack precision and can be written in different ways, Meilisearch does not support facet search on numbers.
```

Comment:
```suggestion
Internally, Meilisearch represents numbers as [`float64`](https://en.wikipedia.org/wiki/Double-precision_floating-point_format). Since they lack precision and can be written in different ways, Meilisearch does not support facet search on numbers. We recommend converting those numbers into strings to make them searchable.
```